R/estimate_KM.R

```diff
@@ -72,13 +72,14 @@ estimate_KM <- function(
 ){
```

Comment:
Can you document the added behaviour to the roxygen section? It would facilitate review as I am not quite sure about the implemented logic although I know the endgame.

Reply:
Once the logic is agreed upon, we need to add UAT for the PR. I can assist with this if required.

Reply:
I added a note about the strata list that is now returned with the object.